src/kohlrahbi/scrape_pruefis/collect_pruefis.py

```diff
@@ -0,0 +1,76 @@
+from datetime import date
+from pathlib import Path
+
+import click
+import docx  # type:ignore[import]
+import tomlkit
+from docx.table import Table  # type:ignore[import]
+from maus.edifact import EdifactFormatVersion
+
+from kohlrahbi.docxfilefinder import DocxFileFinder
+from kohlrahbi.logger import logger
+from kohlrahbi.read_functions import does_the_table_contain_pruefidentifikatoren, get_all_paragraphs_and_tables
+from kohlrahbi.seed import Seed
+
+
+@click.command()
+@click.option(
+    "--format-version",
+    multiple=True,
+    default=[e.value for e in EdifactFormatVersion],  # Set default to all known format versions
+    show_default="All",
+    type=click.Choice([e.value for e in EdifactFormatVersion], case_sensitive=False),
+    help="Format version(s) of the AHB documents. Default is all known format versions.",
+)
+def update_pruefis(format_version):
```

Comment:
kannst du die format-version 
```suggestion
def update_pruefis(format_version: EdifactFormatVersion):
```

Reply:
es ist eher eine liste von EdifactFormatVersion. Habe es aber hier geändert: 1d2ab2bf0717055520c4f1cd53cedbe408d50dd5